hedera-mirror-rest/__tests__/specs/accounts/{id}/rewards/all-params.json

```
@@ -0,0 +1,95 @@
+{
+  "description": "Account staking reward payouts api call with all applicable params",
+  "setup": {
+    "entities": [
+      {
+        "num": 1001
+      },
+      {
+        "num": 800
+      }
+    ],
+    "stakingRewardTransfers": [
+      {
+        "account_id": 1001,
+        "amount": 10,
+        "consensus_timestamp": "1234567890000011111"
+      },
+      {
+        "account_id": 1001,
+        "amount": 12,
+        "consensus_timestamp": "1234567890000022222"
+      },
+      {
+        "account_id": 1001,
+        "amount": 13,
+        "consensus_timestamp": "1234567890000033333"
+      }
+    ]
+  },
+  "tests": [
+    {
+      "url": "/api/v1/accounts/1001/rewards?timestamp=gte:0&order=asc&limit=2",
```

The 3 tests can be combined into one test with 3 URLs
```suggestion
      "urls": [
        "/api/v1/accounts/1001/rewards?timestamp=gte:0&order=asc&limit=2",
        "/api/v1/accounts/0.1001/rewards?timestamp=gte:0&order=asc&limit=2",
        "/api/v1/accounts/0.0.1001/rewards?timestamp=gte:0&order=asc&limit=2"
      ],
```

---

When I try this, 2 of the 3 tests fail.  While all 3 have the same "rewards" (with the account_id in the X.Y.Z format), they have a different "next" link (matching the three different input URL's specification of the account_id).  I'm just calling `utils.getPaginationLink(req, false, last, query.order);` to generate the "next" link; I don't' see a variant that lets me replace the account_id in the input URL with the decoded one.